designer/server/src/routes/forms/create.js

```diff
@@ -103,8 +81,22 @@ export default [
   ({
     method: 'POST',
     path: '/create/title',
-    handler(request, h) {
-      const { payload, yar } = request
+    async handler(request, h) {
+      const { auth, payload, yar } = request
+      const { title } = payload
+      const token = auth.credentials.token
```

Comment:
Tiny little thing, just got a mix of both property access and destructuring here

```suggestion
      const { token } = auth.credentials
```

Reply:
Resolved with 6e6ea6ef